### ISSUE TITLE
Exception: Undefined index: id

### ISSUE THREAD

**Issue by aloupfor** [ecommpro/custom-currency#1]
**Magento version**: 2.3.1
**Plugin version**: 1.0.0

I have the following error when I try to reach Stores > Settings > Configuration:

```
1 exception(s):
Exception #0 (Exception): Notice: Undefined index: id in /app/vendor/magento/module-config/Model/Config/Structure/Element/Iterator.php on line 63
```

It looks like it comes from https://github.com/ecommpro/custom-currency/blob/master/etc/adminhtml/system.xml#L10 as the tab has been removed (https://github.com/ecommpro/custom-currency/commit/211aa74ede0e19d3aada3f981f2dd8c7bf4c65b3).

**Comment by manelio** [comment]
Thank you for your report, @aloupfor. We just fixed the problem with the commit 5176526.

**Comment by aloupfor** [comment]
Thank you for solving the issue.
